### DISCUSSION TITLE
Use host system variables (%HOMEPATH%, etc.) in autoexec?

### DISCUSSION THREAD

**Discussion by emendelson** [joncampbell123/dosbox-x#4426] [Q&A] [answered]
Probably the answer to this is in the wiki, but I can't find it. Is it possible to use variables from the host system in the autoexcec section of a configuration file? For example, this doesn't work (in Windows) and I wonder if there's some way to make it work:

`mount w %HOMEPATH%`

I see that I can start DOISBox-X with a command like this:

`dosbox-x.exe -c "set HOMEPATH=%HOMEPATH%"  -c "mount w %HOMEPATH%" -conf my.conf`

but when I do this, the `set` and `mount` commands don't take effect until after the commands in the autoexec section of the config file have run. The setup I'm working on has an autoexec that ends with IMGMOUNT and then BOOT in order to run Windows 98, so I can't use the command line this way.

Probably I am missing something obvious, and will be grateful for any help.

**Comment by rderooy** [comment, marked as answer]
Have a look at: https://github.com/joncampbell123/dosbox-x/issues/4306

**Reply by emendelson** [reply]
Exactly what I needed. And I should have remembered it from years ago... Thank you.
